renovate.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
how can I test this?

Reply:
I used my private GitHub repositories (and corresponding Konflux builds/quay.io repos) for test:
- https://github.com/TamiTakamiya/content-sample
- https://github.com/TamiTakamiya/service-sample

Reply:
no I meant how can I test this renovate config itself? should I use Renovate CLI?

Reply:
> According to Slack posts like [this](https://redhat-internal.slack.com/archives/C06KTFLDS69/p1739354152407359?thread_ts=1739180700.535179&cid=C06KTFLDS69), renovate, which checks updates of the image, runs every 4 hour. So there might be delays up to 4 hours.

Oh ok just saw your comment.

Reply:
Just one small thing which is not a blocker:

Maybe we can enforce testing the renovate stuff with the CLI instead of waiting Konflux to run it. So we can do our own verification upfront so we can make sure Konflux doesnt fail because of this. WDYT?

Reply:
Probably we can install this and use this locally at least or make the pre-commit run it in the future to validate the config json https://docs.renovatebot.com/getting-started/running/#npm-package-cli

Reply:
I will take a look at it once I have time